src/sections/section7/Section7.jsx

```diff
@@ -1,9 +1,21 @@
+// import Chart from "./Chart";
+
 function Section7() {
   return (
-    <>
-      <p>Section 7</p>
-    </>
+    <div className="container d-flex align-items-center" style={{ height: "100vh" }}>
+      <div className="row">
+        <div className="col-8 mx-auto">
+          <h2 className="mb-2 text-center">Bedeutung des Einsatzes von KI</h2>
```

Comment:
text-center kann raus, damit die Headline wie in Figma gestylet ist; genauso wie mx-auto in Zeile 7 drüber

Reply:
Machen wir dann in einem neuen MR, wenn wir uns um das Layout kümmern 👍